scvi/models/modules.py

```diff
@@ -19,10 +53,21 @@ class FCLayers(nn.Module):
     :param n_layers: The number of fully-connected hidden layers
     :param n_hidden: The number of nodes per hidden layer
     :param dropout_rate: Dropout rate to apply to each of the hidden layers
+    :param n_batch_norm: number of conditional batch normalization (1 is standard)
```

Comment:
What does this number mean ? Can we be more explicit ?

Reply:
I updated the docstring

Reply:
great